lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -77,11 +74,11 @@
         </p>
         <div class="credit-action">
             % if credit_btn_label:
-                <a class="btn credit-btn ${credit_btn_class}" href="${credit_btn_href}" target="_blank" data-course-key="${credit_status['course_key']}" data-user="${user.username}" data-provider="${credit_status['provider_id']}">
-                    ${credit_btn_label}
+                <a class="btn credit-btn ${credit_btn_class | h}" href="${credit_btn_href | h}" target="_blank" data-course-key="${credit_status['course_key'] | h}" data-user="${user.username | h}" data-provider="${credit_status['provider_id'] | h}">
```

Comment:
i don't think credit_btn_class or credit_btn_href need escaping as they are based on literals here in code rather than user data.  username is also probably exempt since we validate usernames strongly according to a whitelist of allowed characters, but i don't mind erring on the side of caution there.